res/css/style.css

```diff
@@ -220,45 +219,52 @@ body, #root, .profileViewer {
   color: transparent;
 }
 
-.treeRowToggleButton {
-  background-image: url(./treetwisty.svg);
-}
-
 .focusCallstackButton {
-  background-image: url(./circlearrow.svg);
+  background-image: url(../img/svg/circlearrow.svg);
   margin-left: 5px;
   visibility: hidden;
 }
 
-.treeRowToggleButton:active:hover,
 .focusCallstackButton:active:hover {
   background-position: -16px 0;
 }
 
-.treeRowToggleButton.collapsed {
-  background-position: 0 -16px;
+.treeViewBody:focus .treeViewBodyInner .treeViewRow.selected > .focusCallstackButton {
```

Comment:
Just wondering if `treeViewBodyInner` is really needed ?

Reply:
yes.

Reply:
I removed it before and it was overwritten. Now it works. Weird. Anyway, removed it.